CollAction/Views/Account/Login.cshtml

```diff
@@ -10,6 +11,8 @@
 
 @{
     ViewData["Title"] = Localizer["Log in"];
+    var consentFeature = Context.Features.Get<ITrackingConsentFeature>();
```

Comment:
Feels like it should be part of the viewmodel.

Reply:
I'm not sure.. I got this from the Microsoft documentation itself. 

Also, there are 3 parts (index, _layout and _cookieConsentPartial) where there is no view model. I could introduce one for the index page, but not for _layout and _cookieConsentPartial. So we'll end up with two ways where this is set.

Reply:
Alright, didn't see that. It's fine as it is then.